application/config/config.go

```diff
@@ -61,6 +61,7 @@ const (
 	FormatMd              = "md"
 	snykCodeTimeoutKey    = "SNYK_CODE_TIMEOUT" // timeout as duration (number + unit), e.g. 10m
 	DefaultSnykApiUrl     = "https://snyk.io/api"
```

Comment:
Not necessary for this PR, but should probably be changed to `https://api.snyk.io`